core-bundle/src/Resources/contao/templates/modules/mod_breadcrumb.html5

```diff
@@ -2,14 +2,16 @@
 
 <?php $this->block('content'); ?>
 
+<nav aria-label="Breadcrumb">
```

Comment:
This is inconsistent with the other navigation modules. The markup of the regular navigation module for example is:

```html
<nav class="mod_navigation block">
  …
</nav>
```

whereas the breadcrumb would now be:

```
<div class="mod_breadcrumb block">
  <nav aria-label="Breadcrumb">
    …
  </nav>
</div>
```

To be consistent `mod_breadcrumb` should not extend from `block_unsearchable` and implements its wrapper directly instead, while also implementing the same blocks as `block_unsearchable`. This will make this change less intrusive, otherwise we might break a lot of Contao instances in a bugfix update.

Reply:
Updated. I also made sure to put `aria-label` first so it's easier to merge upstream where we removed the schema.org markup and moved it to JSON.